pkg/apis/application/v1alpha1/zz_generated.openapi.go

```diff
@@ -179,6 +178,59 @@ func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenA
 		"github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1.TagFilter":                               schema_pkg_apis_application_v1alpha1_TagFilter(ref),
 		"github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1.objectMeta":                              schema_pkg_apis_application_v1alpha1_objectMeta(ref),
 		"github.com/argoproj/argo-cd/v2/pkg/apis/application/v1alpha1.rawResourceOverride":                     schema_pkg_apis_application_v1alpha1_rawResourceOverride(ref),
+		"k8s.io/apimachinery/pkg/apis/meta/v1.APIGroup":                                                        schema_pkg_apis_meta_v1_APIGroup(ref),
```

Comment:
Not sure if I should be concerned that a bunch of meta stuff was added to the schema.